### PR TITLE
refactor: [M3-8710] - Move `Notice` & `Tooltip` components to UI package and update imports

### PR DESCRIPTION
**Pull request by hkhalil-akamai**
## Description 📝
Moves the `Notice` component, story and unit tests to the `@linode/ui` package and updates existing imports.

## Changes  🔄
- Copy `Notice` component to `@linode/ui` package
- Move needed icons to `ui/src/assets/icons` directory
- Copied `renderWithTheme` test helper into UI package
- Update existing imports

## How to test 🧪

* Verify tests pass
* Verify Notices continue to render as expected
   * Check that Notices with icons (`important={true}`) render with the expected SVG
     * For example, enable MSW with the Legacy present and verify the icon renders in the maintenance banner

### CODE REVIEW

**Comment by hkhalil-akamai** [comment]
Lots of conflicts generated due to number of affected files and reformatting. I will address those once the PR is approved and ready to merge.

**Comment by coliu-akamai** [comment]
image

Seems like the storybook and test checks are failing bc of a missing file?

**Review by harsh-akamai — APPROVED** [review]
Changes look good! Thanks @hkhalil-akamai

File: packages/manager/src/components/TabbedPanel/TabbedPanel.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
+import { Notice } from '@linode/ui';
 import { Tooltip } from '@linode/ui';
```
[the remark this answers is not in the document]
Comment (harsh-akamai) [reply packages/manager/src/components/TabbedPanel/TabbedPanel.tsx:2]: This PR is missing a changeset.

**Review by dwiley-akamai — APPROVED** [review]
Code review ✅
Cloud looks good, no `Notice` regressions observed ✅

Approved pending passing CI tests & resolved conflicts.

<details>
<summary>Observed some unit test failures locally</summary>

Screenshot 2024-11-05 at 2 16 25 PM
</details>

**Comment by hkhalil-akamai** [comment]
> Observed some unit test failures locally

I'm getting some flake, but unable to consistently replicate any unit test failures locally

**Comment by hkhalil-akamai** [comment]
Bypassing merge protections to avoid further merge conflicts. Except for some flake, all unit and integration tests have passed previously.

**Pull request merged**